### ISSUE TITLE
[Bug] cannot manual edit date on create transaction

### ISSUE THREAD

**Issue by a-johanes** [jameskokoska/Cashew#1060]
**Describe the bug**
when creating a new transaction, cannot manually edit the date as the keyboard only shows number

**How to reproduce**
1. create a new transaction
2. edit the date
3. click pencil for manual edit
4. cannot edit the string

**Screenshots/videos**

<img width="591" height="1152" alt="Image" src="https://github.com/user-attachments/assets/22d71222-c252-44bf-a4b1-f08e26b3ac45" />

<img width="591" height="1127" alt="Image" src="https://github.com/user-attachments/assets/7271c10a-3aa3-470b-9085-0b84b639350c" />

**Details**
 - Device: Samsung Galaxy S21 FE 5G
 - App Version: v6.2.7+470, db-v48

NOTE TO THE SUBMITTER
Reporting bugs is important for maintaining the app. Thank you for helping make the app a better experience for all!

**Comment by a-johanes** [comment]
sorry duplicate of #426. anyway for reference, I am using samsung keyboard
